site/en/docs/privacy-sandbox/demos/index.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Privacy Sandbox demos'
+subhead: >
+  Demos and colabs walk you through the Privacy Sandbox APIs.
+description: >
+  Demos and colabs walk you through the Privacy Sandbox APIs.
+date: 2023-05-29
+authors:
+  - alexandrawhite
+---
+
+There are a number of demos available for the Privacy Sandbox APIs. Most of them will require you to be familiar with the APIs, so if you haven't read the docs, check out the links we've listed along with the demos here.
+
+<p>
+
+</p>
+
+## Demos and colabs
+
+{% Details %}
+{% DetailsSummary %}
+### Protected Audience API (FLEDGE)
+{% endDetailsSummary %}
+
+{% Columns %}
+{% Column %}
+This demo uses FLEDGE to join ad interest groups on two advertiser sites,
+and then initiates an on-device auction to select an ad for display on a publisher site.
+
+[Demo](https://fledge-demo.glitch.me/)
```

Comment:
https://protected-audience-demo.web.app/ is now the new main demo for PAAPI

Reply:
Oh thanks!